Essentials/src/main/java/com/earth2me/essentials/User.java

```diff
@@ -1242,4 +1242,9 @@ public boolean isToggleShout() {
         }
         return toggleShout == null ? toggleShout = ess.getSettings().isShoutDefault() : toggleShout;
     }
+
+    public boolean isWhitelisted() {
```

Comment:
We don't need a new method here, just use user.getBase().isWhitelisted() in /seen and /whois